src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -21,8 +21,8 @@ export const PERFORMANCE_EVENT_VIEW: Readonly<NewQuery> = {
     'p95()',
     'error_rate()',
     'apdex(300)',
-    'user_misery(300)',
     'count_unique(user)',
+    'user_misery(300)',
```

Comment:
Should we omit count_unique from the rendered table for performance now?

Reply:
I just feel like the scorebar isn't useful without some sense of the magnitude. If it just showed "5/10", does that mean only 1 out of 2 users are miserable? Or 20,000 out 40,000? I know the tooltip exposes that, but it's not great to hover over each column to get that information.

Reply:
Fair.